serverless/src/getContactInfo/handler.js

```diff
@@ -31,18 +34,41 @@ const getContactInfo = async (event, context) => {
   try {
     const userRecord = await dbConnection('users')
       .first(
-        'cmr_preferences',
+        'urs_id',
         'urs_profile'
       )
       .where({
         id
       })
 
+    const {
+      access_token: authToken
+    } = await getAccessTokenFromJwtToken(jwtToken, earthdataEnvironment)
+
+    // Make a request to CMR-ordering instead of the EDSC database
+    const cmrPreferencesData = await getCmrPreferencesData(
+      userRecord.urs_id,
+      authToken,
+      earthdataEnvironment
+    )
+    const { status, data } = cmrPreferencesData
+
+    const { errors } = data
+
+    if (errors) throw new Error(JSON.stringify(errors))
+
+    const { user } = data.data
```

Comment:
We try to avoid chaining like this (`data.data`), what we would typically do is rename `data` on line 54 to `responseData`, then on line 56 pull `errors` and `data` out of `responseData`, then this line becomes `const { user } = data`